docs/en/advanced-concepts/multiple-xcode-versions.md

```diff
@@ -0,0 +1,31 @@
+## Running appium with multiple Xcode versions installed
```

Comment:
I'd also change appium to Appium everywhere

Reply:
fixed